setup.py

```diff
@@ -37,6 +37,12 @@
 # Always include standard site-packages to PYTHONPATH
 os.environ['PYTHONPATH'] = sysconfig.get_paths()['purelib']
 
+# Produce static Rust binaries.
+#
+# This is required to use the same virtme-ng-init across different root
+# filesystems (when `--root DIR` is used).
+os.environ["RUSTFLAGS"] = "-C target-feature=+crt-static"
```

Comment:
Just to be sure, is it fine to override the `RUSTFLAGS` variable? Or should it be extended?

Reply:
Good point @matttbe, it's probably safer to extend it, I'll update the PR.